test/hw7/test_task3.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+from All_home_works.hw7.task3 import tic_tac_toe_checker
+
+
+@pytest.mark.parametrize(  # noqa: PT006
+    "input_data, expected",
+    [
+        ([["-", "-", "o"], ["-", "o", "o"], ["x", "x", "x"]], "x wins!"),
+        ([["o", "o", "o"], ["x", "x", "o"], ["x", "o", "x"]], "o wins!"),
+    ],
+)
+def test_tic_tac_toe_checker__return_win_in_row(input_data, expected):
+    assert expected == tic_tac_toe_checker(input_data)
+
+
+@pytest.mark.parametrize(  # noqa: PT006
```

Comment:
What is PT006?

Reply:
https://github.com/E1-Bart0/homework-repo/pull/25#discussion_r627397514